ingestion/src/metadata/utils/airbyte_client.py

```diff
@@ -0,0 +1,68 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+Client to interact with airbyte apis
+"""
+import json
+from typing import List
+
+from metadata.generated.schema.entity.services.connections.pipeline.airbyteConnection import (
+    AirbyteConnection,
+)
+from metadata.ingestion.ometa.client import REST, APIError, ClientConfig
+
+
+class AirbyteClient:
+    def __init__(self, config: AirbyteConnection):
+        self.config = config
+        client_config: ClientConfig = ClientConfig(
+            base_url=self.config.hostPort,
+            api_version="api/v1",
+            auth_header="Authorization",
+            auth_token=lambda: ("no_token", 0),
+        )
+        self.base_url = config.hostPort + "/api/v1"
+        self.headers = {
+            "Authorization": "Bearer no_token",
+            "Content-Type": "application/json",
+        }
```

Comment:
`self.base_url` and `self.headers` can be removed from here since it is not being used anywhere